lib/web_ui/lib/src/engine/html/transform.dart

```diff
@@ -14,17 +22,14 @@ class PersistedTransform extends PersistedContainerSurface
 
   @override
   void recomputeTransformAndClip() {
-    _transform = parent!._transform!.multiplied(Matrix4.fromFloat32List(matrix4));
-    _localTransformInverse = null;
-    _projectedClip = null;
+    transform = parent!.transform!.multiplied(Matrix4.fromFloat32List(matrix4));
+    localTransformInverse = null;
+    projectedClip = null;
   }
 
   @override
-  Matrix4? get localTransformInverse {
-    _localTransformInverse ??=
-        Matrix4.tryInvert(Matrix4.fromFloat32List(matrix4));
-    return _localTransformInverse;
-  }
+  Matrix4? get defaultLocalTransformInverse =>
```

Comment:
losing local caching here.

Reply:
We are not losing caching because the caching is handled by the grandparent `PersistedSurface` class.

(anyway, this has changed so the comment doesn't apply anymore)